blocks/hotspot/hotspot.js

```diff
@@ -0,0 +1,53 @@
+export default function decorate(block) {
+  [...block.children].forEach((row, r) => {
+    if (r > 0) {
+      const content = [...row.children][0].textContent.trim();
+      const isImage = content.endsWith('.jpg') || content.endsWith('.png') || content.endsWith('.gif') || content.endsWith('.jpeg');
```

Comment:
There could be images even without these extensions, for e.g. https://delivery-p66302-e574366.adobeaemcloud.com/adobe/assets/urn:aaid:aem:6928dc02-87be-4a00-93d5-7b3284f22e09

Reply:
Any image URL can be incorporated as per the requirement. We can use also use regex as per requirement which is upto customisation.

Reply:
yes but I am wondering if there's no way of telling the image by just looking at the URL. One option could be to express it as a variant of the block. For e.g. `Hotspot(image)` , `Hotspot(video)` etc.  and based on that, decide the tag to inject.

Reply:
That will work but then wouldn't it limit the block to either have a specific type of hotspot content (either image or vide )and not a mix of contents.

Reply:
But yes, I will make the changes for fixed hotspot content.